src/modules/apps/e2e/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { Color, render } from 'ink'
+
+import log from '../../../logger'
+import { apps, tester } from '../../../clients'
+import { getManifest, writeManifestSchema } from '../../../manifest'
+import { toAppLocator } from '../../../locator'
+import RealTimeReport from './report'
+import { getToken } from '../../../conf'
+
+class ErrorBoundary extends React.Component<{}, { error: any }> {
+  constructor(props) {
+    super(props)
+    this.state = { error: null }
+  }
+
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+    return { error }
+  }
+
+  componentDidCatch(_error, _errorInfo) {
+    // You can also log the error to an error reporting service
+    // logErrorToMyService(error, errorInfo);
+  }
+
+  render() {
+    if (this.state.error != null) {
+      // You can render any custom fallback UI
+      return <Color red>{this.state.error.message}</Color>
+    }
+
+    return this.props.children
+  }
+}
+
+const TEST_WORKSPACE = ''
+
+const resolveAppLocator = async options => {
+  if (options.workspace) return TEST_WORKSPACE
+  const manifest = await getManifest()
+  try {
+    await writeManifestSchema()
+  } catch (e) {
+    log.debug('Failed to write schema on manifest.')
+  }
+  return toAppLocator(manifest)
+}
+
+export default async options => {
+  // await validateAppAction('e2e')
```

Comment:
This can be removed

Reply:
do you mean to change the default, the comment or the whole options parameter?

Reply:
Ops, sorry, the comment